providers/celery/src/airflow/providers/celery/executors/celery_executor_utils.py

```diff
@@ -125,21 +126,54 @@ def on_celery_import_modules(*args, **kwargs):
         import kubernetes.client  # noqa: F401
 
 
-@app.task
-def execute_command(command_to_exec: CommandType) -> None:
-    """Execute command."""
-    dag_id, task_id = BaseExecutor.validate_airflow_tasks_run_command(command_to_exec)
+# Once Celery5 is out of beta, we can pass `pydantic=True` to the decorator and it will handle the validation
+# and deserialization for us
+@app.task(name="execute_workload")
```

Comment:
This is a small thing, but by specifying the name here, it means this is what shows up in the Celery logs, rather than `airflow.providers.celery.executors.celery_executor_utils.execute_workload`!

Reply:
Cool, damn nice!